metrics.go

```diff
@@ -47,6 +47,12 @@ var (
 		},
 		nil,
 	)
+	metricDeviceCount = prometheus.NewDesc(
+		"smartctl_device_count",
```

Comment:
`_count` is a reserved convention for histograms. This should be `"smartctl_devices"`.